aut-apps.sh

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root" 1>&2
+   exit 1
+fi
+echo 'Install Google Chrome, Git and Visual Studio Code'
+touch /etc/apt/sources.list.d/google-chrome.list
+echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main'>/etc/apt/sources.list.d/google-chrome.list
+wget https://dl.google.com/linux/linux_signing_key.pub
```

Comment:
Instead of saving the key on disk, pipe it right into the apt-key command. You can use `-O -` arg to do so.